vendor-extra/JatsContentBundle/src/EventListener/BuildView/ItemTeaserListener.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener\BuildView;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\SimplifiedViewConverterListener;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use function Libero\ViewsBundle\array_has_key;
+
+final class ItemTeaserListener
+{
+    use SimplifiedViewConverterListener;
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    protected function handle(Element $object, View $view) : View
+    {
+        $heading = $object->ownerDocument->xpath()
+            ->firstOf(
+                '/libero:item/jats:article/jats:front/jats:article-meta/jats:title-group/jats:article-title',
+                $object
+            );
+
+        if (!$heading instanceof Element) {
+            return $view;
+        }
+
+        return $view
+            // @todo - set route to content.
+            ->withArgument('href', '#')
```

Comment:
We will need to access the routes here.

Reply:
This will be something similar to https://github.com/elifesciences/journal/blob/develop/src/Router/ModelParameterResolver.php (rather than 'create this route for me', it's 'give me the path for this piece of content').

Will have a think.